config/300-customrun.yaml

```diff
@@ -0,0 +1,66 @@
+# Copyright 2020 The Tekton Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  name: customruns.tekton.dev
+  labels:
+    app.kubernetes.io/instance: default
+    app.kubernetes.io/part-of: tekton-pipelines
+    pipeline.tekton.dev/release: "devel"
+    version: "devel"
+spec:
+  group: tekton.dev
+  preserveUnknownFields: false
+  versions:
+  - name: v1beta1
+    served: true
```

Comment:
want to double check that it's intentional that we're serving this CRD? would it make sense to resolve the questions around retries + retriesStatus first?

Reply:
Good point - I'll change this to false.

Reply:
another approach could be that we don't support retries in customruns in the initial release -- until we agree on a way forward in [TEP-0121](https://github.com/tektoncd/community/pull/816 )-- what do you think? happy to open PR updating [TEP-0114](https://github.com/tektoncd/community/blob/main/teps/0114-custom-tasks-beta.md) 

cc @abayer @lbernick @XinruZhang

Reply:
Updated TEP-0114: https://github.com/tektoncd/community/pull/862

Reply:
Yeah, I think we're going to need to do that, because changing to `served: false` broke all the e2e tests. =)